cassandra/1.cql

```diff
@@ -12,7 +12,8 @@ CREATE TABLE devgym.task (
   year TIMESTAMP,
   type TEXT,
   timeuuid TIMEUUID,
-  task_description TEXT,
+  name TEXT,
```

Comment:
Ok. http://codingbat.com there are names as well. I think they make sense for now.